quick-start.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+################################################################################
+#
+# This script is used to setup gcloud authentication for GitHub Actions.
+# It is based on the following guide: https://github.com/google-github-actions/auth#setup
+#
+################################################################################
+
+# Prompt user for project name and repository name
+echo "Welcome to the PHDI Google Cloud setup script!"
+echo "This script will help you setup gcloud authentication for GitHub Actions."
+echo "We need some info from you to get started."
+echo "After entering the info, a browser window will open for you to authenticate with Google Cloud."
+echo
+while true; do
+    read -p "Do you already have a Project in Google Cloud Platform? (y/n) " yn
+    case $yn in
+        [Yy]* ) read -p "Please enter the name of the existing Project. " PROJECT_NAME; NEW_PROJECT=false; break;;
+        [Nn]* ) read -p "Please enter a name for a new Project. " PROJECT_NAME; NEW_PROJECT=true; break;;
+        * ) echo "Please answer y or n.";;
+    esac
+done
+echo "Please enter the name of the repository you will be deploying from."
+echo "For example, if your repo URL is https://github.com/CDCgov/phdi-google-cloud, enter: CDCgov/phdi-google-cloud"
+read GITHUB_REPO
+
+# Login to Google Cloud Platform
+gcloud auth login
+
+# Create a project if needed and get the project ID
+if [ $NEW_PROJECT = true ]; then
+    gcloud projects create $PROJECT_NAME
+fi
+PROJECT_ID=$(gcloud projects list --filter="name:${PROJECT_NAME}" --format="value(projectId)")
+if [ -z "$PROJECT_ID" ]; then
+    echo "Error: Project ID not found. To list projects, run 'gcloud projects list'."
+    exit 1
+fi
+
+# Create a service account
+gcloud iam service-accounts create "github" \
+  --project "${PROJECT_ID}" \
+  --display-name "github"
+
```

Comment:
What level of permissions does a user need to create service accounts with sufficient access for deploying Terraform. Should we document this as a requirement for running this script?